frontend/src/main/components/RiderApplication/RiderApplicationForm.js

```diff
@@ -32,16 +35,16 @@ function RiderApplicationForm({ initialContents, submitAction, buttonLabel = "Ap
                         type="text"
                         {...register("id")}
                         defaultValue={initialContents?.id}
-                        disabled
+                        disabled={disableBool}
                     />
                 </Form.Group>
-            )}
+            )} */}
```

Comment:
I assume this is commented because you don't want the id shown to the user when they click show or edit, but either remove or keep it in. I'd rather you keep this field in the form, and if you don't want it to show, find some other way of achieving that. Its also fine to show it since the application id also shows up in the ride request table anyways

Reply:
I'll keep the Application Id field in then